docs/source/getting-started/tabular-data/computing_fundemental_statistics.ipynb

```diff
@@ -0,0 +1,473 @@
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Actual Total Hours:  2962104.0\n"
+     ]
+    }
+   ],
+   "source": [
+    "total_hours_actual = df.select(pl.col(\"HWUSUAL\").sum()).collect().item()\n",
+    "print('Actual Total Hours: ', total_hours_actual)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The query for the sum is essentially the same. We add an extra argument, `fill_null` because the `sum` method requires a non-nullable input. \n",
+    "\n",
+    "Do not compute the values you use to impute null values or specify the bounds since doing so will hinder the privacy guarantee. These values should be based on domain knowledge. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Differentially Private Total Hours:  2953689.33103334\n"
+     ]
+    }
+   ],
+   "source": [
+    "total_hours_dp = context.query().select(pl.col(\"HWUSUAL\").fill_null(0.).dp.sum((1,80))).release().collect().item()\n",
+    "print('Differentially Private Total Hours: ', total_hours_dp)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Mean"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To demonstrate the `mean` method, calculate the mean number of hours worked for all years in the entire dataset. The `mean` method also requires all null values to be filled. The bounds parameter is the same as the bounds parameter used in `sum`. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Actual Hours Mean:  37.63409056258576\n"
+     ]
+    }
+   ],
+   "source": [
+    "mean_hours_actual = df.select(pl.col(\"HWUSUAL\").mean()).collect().item()\n",
+    "print('Actual Hours Mean: ', mean_hours_actual)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Differentially Private Mean for Hours:  37.72491125272871\n"
+     ]
+    }
+   ],
+   "source": [
+    "mean_hours_dp = context.query().select(pl.col(\"HWUSUAL\").fill_null(40.).dp.mean((1,80))).release().collect().item()\n",
+    "print('Differentially Private Mean for Hours: ', mean_hours_dp )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Median\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To demonstrate the `median` method, calculate the median number of hours worked for all years in the entire dataset. The `median` method requires a parameter `candidates,` which is the potential value for the median. "
```

Comment:
```suggestion
    "To demonstrate the `median` method, calculate the median number of hours worked for all years in the entire dataset. The `median` method requires a parameter `candidates,` which are potential values for the median. "
```

Do you discuss anywhere the trade-offs between having more or fewer candidates? 1 hour increments are essentially arbitrary.